### ISSUE TITLE
Support local storage for uploading assets

### ISSUE THREAD

**Issue by manhtai** [memoetapp/memoet#50]

**Comment by fluffystranger** [comment]
Hello,
I enjoy using what you've build here very much, so many thanks for sharing this with the community at large.
I would really like to switch to hosting my uploaded assets (e.g. pictures) locally rather than an AWS bucket/S3.

I'm not familiar with how the elixir/waffle implementation fully works.
Would adapting [config.exs](https://github.com/memoetapp/memoet/blob/master/config/config.exs#L53) to follow the local example below be enough, or is a more complex change needed?
https://github.com/elixir-waffle/waffle/blob/master/documentation/examples/local.md

**Comment by manhtai** [comment]
Yes it would be a simple change. Plz try it yourself first then I will make the change later.

**Comment by SONDLecT** [comment]
Just checking on the status of this issue. It sounds like it was an easy fix, I’d definitely like to have this feature. I’m working on a Firefox browser extension for Memoet that would convert anki decks (and hopefully streamline the adding of new cards/decks otherwise), and it would make things simpler if there was a direct way of handling the potentially bundled media files by uploading them locally instead of to an Amazon bucket
